script/tool/lib/src/license_check_command.dart

```diff
@@ -0,0 +1,162 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+
+import 'package:file/file.dart';
+import 'package:path/path.dart' as p;
+
+import 'common.dart';
+
+const Set<String> _codeFileExtensions = <String>{
+  '.c',
+  '.cc',
+  '.cpp',
+  '.dart',
+  '.h',
+  '.java',
+  '.m',
+  '.mm',
+  '.swift',
+  '.sh',
```

Comment:
Needs `.html`? I'm sure our example/web/index.html files don't have a license :/ (I can add them later)

Reply:
Ah, good call. Will add right now and fix the files (what's another 13 files in this PR)